pkg/runner/runner_test.go

```diff
@@ -89,6 +89,7 @@ func TestRunEvent(t *testing.T) {
 		{"testdata", "basic", "push", "", platforms, ""},
 		{"testdata", "fail", "push", "exit with `FAILURE`: 1", platforms, ""},
 		{"testdata", "runs-on", "push", "", platforms, ""},
+		{"testdata", "checkout", "push", "", platforms, ""},
```

Comment:
do we really need that?
![image](https://user-images.githubusercontent.com/31106839/117546541-325d7500-b02b-11eb-97eb-17e6799a4796.png)

Reply:
I think it's important for it to be specifically called out as opposed to being a side-effect of other tests. It's like saying do we really need basic because the other commands run shell scripts too?